libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -1181,7 +1200,8 @@ void Compass::_probe_external_i2c_compasses(void)
     const uint8_t rm3100_addresses[] = { HAL_COMPASS_RM3100_I2C_ADDR1,
                                          HAL_COMPASS_RM3100_I2C_ADDR2,
                                          HAL_COMPASS_RM3100_I2C_ADDR3,
-                                         HAL_COMPASS_RM3100_I2C_ADDR4 };
+                                         HAL_COMPASS_RM3100_I2C_ADDR4
+                                       };
```

Comment:
Undo all the whitespace fixes in the file and Compass.h

We try to keep the commit history only to functional changes. (Feel free to fix the whitespace if you the line though ;) )

```suggestion
                                         HAL_COMPASS_RM3100_I2C_ADDR4
                                       };
```